src/inverseRobot.js

```diff
@@ -7,7 +7,18 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const checkArr = Object.values(robot);
+  const res = {};
+
+  if (checkArr.length !== new Set(checkArr).size) {
+    return null;
+  }
+
+  for (const key in robot) {
+    res[robot[key]] = key;
```

Comment:
You need to create a variable for robot[key]

Reply:
Thank you for your comment. I have optimized my solution!